### DISCUSSION TITLE
Docker restore backup command

### DISCUSSION THREAD

**Discussion by Mondmarmelade** [buanet/ioBroker.docker#363] [Questions & Answers / Fragen & Antworten] [answered]
Hello!
I am new to ioBroker (and also Docker a bit) and I wanted to ask if it is somehow possible to restore a backup from the command line.
I can of course do it manually with the backitup extension, but this requires me to go through the setup process again.
This is very tedious, so is there a workaround?
Thanks in advance!

**Comment by winnyschuster** [comment, marked as answer]
in console,
* `m on -y` to stop iobroker first, if not using iobrokers standard container, use
* `pkill -u iobroker`
* `cd /opt/iobroker` if not already there. find the name of your backup with 
* `ls -ltr backups|tail -4` which gives you the 4 latest files, copy the one youd like to use
* `iobroker restore file_to_restore`
* `m off -y` once you see the message that everything was restored successfully

last command will restart container, otherwise do this manually. after short time you should be able to access adminGUI.

**However, you need to wait until all adapters are installed, you could see this in instances tab** depending on amount of adapters and your system that can easily take 30 minutes. you will need to start your adapters **manually** once everything is finished

**Comment by Mondmarmelade** [comment]
Wait?
How can I use `cd` in the docker container?

**Comment by winnyschuster** [comment]
> Wait? How can I use `cd` in the docker container?

that is depending on your system and how you startet your container. Normally all NAS have a GUI that offers a console option to access a running containers console

generally, from the Docker hosts console, enter ```docker exec -t your_container_name /bin/bash```
